runtimes/pythonrt/pythonrt.go

```diff
@@ -76,6 +76,7 @@ func (py *pySvc) cleanup(ctx context.Context) {
 	if err := runnerManager.Stop(ctx, py.runnerID); err != nil {
 		py.log.Warn("close runner", zap.Error(err))
 	}
+	py.runner.Close()
```

Comment:
See below about returning error, log it here.

Reply:
done